pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -13,11 +13,11 @@ let
   buildKernel = any (n: n == configFile) [ "kernel" "all" ];
   buildUser = any (n: n == configFile) [ "user" "all" ];
 
-  common = { version, sha256, extraPatches, spl, inkompatibleKernelVersion ? null } @ args:
+  common = { version, sha256, extraPatches, spl, incompatibleKernelVersion ? null } @ args:
     if buildKernel &&
-       (inkompatibleKernelVersion != null) &&
-       versionAtLeast kernel.version inkompatibleKernelVersion then
-      throw "linux v${kernel.version} is not yet supported by zfsonlinux v${version}"
+       (incompatibleKernelVersion != null) &&
+       versionAtLeast kernel.version incompatibleKernelVersion then
+      throw "Linux v${kernel.version} is not yet supported by zfsonlinux v${version}. Try zfsUnstable or set the NixOS option boot.zfs.enableUnstable."
```

Comment:
this error message might become a problem, once unstable is also not compatible with mainline...